test/integration/sap_hana_ha_simple_example/controls/deployment_validation.rb

```diff
@@ -28,22 +14,11 @@
 
 control 'deployment_validation_primary' do
 
-  describe command("gcloud compute instances get-serial-port-output #{attribute('primary_instance_name')} --project=#{attribute('project_id')} --zone=#{attribute('primary_zone')}") do
+  describe command("sleep 100 ; gcloud compute instances describe #{attribute('primary_instance_name')} --project=#{attribute('project_id')} --zone=#{attribute('primary_zone')}") do
       its(:exit_status) { should eq 0 }
 
-      context "output of df -h command" do
-        its('stdout') { should match('/dev/mapper/vg_hana-data') }
-        its('stdout') { should match('/dev/mapper/vg_hana-log') }
-      end
-
-      context "output of HDB info command" do
```

Comment:
Are these tests no longer useful/correct?

Reply:
The SAP HANA startup scripts were recently modified to add a "status=complete" metadata once the setup has been completed. So I modified the test case to check that, which is more future-proof, as it does not depend in log lines that can easily change..